### PR TITLE
README: add some contributing rules about code style

### PR DESCRIPTION
**Pull request by Morriar**
Just trolling @privat.

Signed-off-by: Alexandre Terrasa alexandre@moz-code.org

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,3 +39,16 @@ make populate
...
+The `Missions` team is happy to receive contributions and suggestions.
+Give a look at the Github repo
+[https://github.com/Morriar/Missions](https://github.com/Morriar/Missions).
+
+### Code style
+
+Mission enforces the following code style rules:
+
+* All the Nit coding standards
+* The `do` is on the same line than the function declaration
```
Comment (lbajolet) [inline README.md]: Oh, and :s/than/as

**Comment by lbajolet** [comment]
+1

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Morriar) [reply README.md]: Damn I **always** do this error...

[the remark this answers is not in the document]
Comment (lbajolet) [reply README.md]: You **SHOULD** consider buying one of [these](http://www.dummies.com/education/language-arts/grammar/english-grammar-for-dummies-cheat-sheet/)

**Pull request merged**